### ISSUE TITLE
Spotify no longer surfaces their spotify URI Link

### ISSUE THREAD

**Issue by reduxdj** [hnarayanan/shpotify#154]
No Longer:
<img width="1448" alt="Screen Shot 2021-10-27 at 7 07 24 AM" src="https://user-images.githubusercontent.com/322547/139083935-d4f2a6af-c600-4665-a41c-567135de6778.png">

Now:
<img width="561" alt="Screen Shot 2021-10-27 at 7 05 29 AM" src="https://user-images.githubusercontent.com/322547/139084039-09797b3d-d8f0-4364-a283-fb51dc197c92.png">

**Comment by davidem** [comment]
No, api calls and applescript still provide and handle uri's, fortunately 😁

**Comment by davidem** [comment]
By the way: I just found out that uri's can still be displayed in the gui, by pressing alt/ctrl. 

<img width="393" alt="image" src="https://user-images.githubusercontent.com/706591/165599236-0f24de33-b7ad-4808-916a-de3ab2a9252e.png">

As mentioned here: https://community.spotify.com/t5/FAQs/What-s-a-Spotify-URI/ta-p/919201

🤓

**Comment by hnarayanan** [comment]
Detective work paying off!
